ripley/api/status_controller.py

```diff
@@ -112,9 +135,8 @@ def _db_status():
 
 def _ping_calnet():
     try:
-        uid = '1022796'
```

Comment:
When this person retires, we won't need a Nagios alert to let us know about it.

Reply:
1022796 should hurry up and retire before this PR is merged.